src/main/java/com/mongodb/kafka/connect/sink/MongoSinkConfig.java

```diff
@@ -193,7 +192,6 @@ private static ConfigDef createConfigDef() {
           @SuppressWarnings("unchecked")
           public Map<String, ConfigValue> validateAll(final Map<String, String> props) {
             Map<String, ConfigValue> results = super.validateAll(props);
-            IGNORED_CONFIGS.forEach(c -> results.remove(c));
```

Comment:
This was added to fix a bug: [https://jira.mongodb.org/browse/KAFKA-166](https://jira.mongodb.org/browse/KAFKA-166).
By removing this aren't we re-introducing that bug?

Also, [this](https://github.com/confluentinc/mongo-kafka/commit/3085c9cb1ff4f943341fa29da43d1cc2976e5f5c) commit also made changes to MongoSinkTopicConfig.java
Have we verified it is not breaking anything?

Reply:
Hey @rajdangwal, the config in conflict in the ticket is https://github.com/mongodb/mongo-kafka/blob/2f32a3baf0f1be2cafc598c4356612c46b736b55/src/main/java/com/mongodb/kafka/connect/sink/MongoSinkTopicConfig.java#L85 `topic` vs `topics` coming from kafka connect.
The change I'm making is to https://github.com/mongodb/mongo-kafka/blob/2f32a3baf0f1be2cafc598c4356612c46b736b55/src/main/java/com/mongodb/kafka/connect/sink/MongoSinkConfig.java#L76 `topic.override.%s.%s`.
It was being ignored during validation, and now its configDef is returned with /validate requests.
I'm not introducing this config to UI. It was shown previously as well, ref -> https://docs.mongodb.com/kafka-connector/v1.4/kafka-sink-properties/#topic-specific-configuration-settings

Reply:
Thanks @sudeshwasnik 
Here is the commit that was added to fix the bug for topic vs topics: [https://github.com/mongodb/mongo-kafka/commit/3085c9cb1ff4f943341fa29da43d1cc2976e5f5c](https://github.com/mongodb/mongo-kafka/commit/3085c9cb1ff4f943341fa29da43d1cc2976e5f5c)
That commit introduced the changes (about topic.override.%s.%s) which you are removing in your commit. 

I don't think you are getting the error message on hitting the validate endpoint with your changes, as described [here](https://jira.mongodb.org/browse/KAFKA-166). Is that correct?
Maybe that was a bug from Kafka side which got fixed. Can you confirm?
If no error is coming, then these changes LGTM. 
Thanks.